README.md

```diff
@@ -59,6 +59,15 @@ $ flutter pub get
         * [Installing](../main/docs/TOOLTIP.md#installing-hammer_and_wrench)
         * [How to use](../main/docs/TOOLTIP.md#how-to-use-building_construction)
         * [Attributes](../main/docs/TOOLTIP.md#attributes-gear)
+    * [AppSwitch](../main/docs/SWITCH.md)
+        * [How to use](../main/docs/SWITCH.md#how-to-use-building_construction)
+        * [Attributes](../main/docs/SWITCH.md#attributes-gear)
+    * [AppRaisedButton](../main/docs/BUTTON.md)
+        * [How to use](../main/docs/BUTTON.md#how-to-use-building_construction)
+        * [Attributes](../main/docs/BUTTON.md#attributes-gear)
+    * [AppCard](../main/docs/CARD.md)
+        * [How to use](../main/docs/CARD.md#how-to-use-building_construction)
+        * [Attributes](../main/docs/CARD.md#attributes-gear)
```

Comment:
❓  I've tried to tap on few of them and they all seem to lead to `404 Page Not Found `, is it working for you ? :)

Reply:
Okay I can see the files in the `docs` directory of your branch. 
The issue (not sure if it actually is) is that the link brings to the docs directory of the `main` branch, where, of course, the files are not there yet :)

Reply:
it will work, when it will be merged to the `main` branch 😄